### ISSUE TITLE
HighResolutionTimeStamp in milliseconds

### ISSUE THREAD

**Issue by JIoJIaJIu** [servo/servo#5814]
[In specification](https://dvcs.w3.org/hg/webperf/raw-file/tip/specs/HighResolutionTime/Overview.html#sec-DOMHighResTimeStamp)

> A DOMHighResTimeStamp SHOULD represent a number of milliseconds accurate to a thousandth of a millisecond.

But in [code](https://github.com/servo/servo/blob/1b08211a5e9ec00eeaf241b916d51dc665289825/components/script/dom/performance.rs#L54) it looks like -10^15
```rust
let now = (time::precise_time_ns() as f64 - navStart) * 1000000 as f64;
```

**Comment by jdm** [comment]
This is fixed by #5697, right? I previously filed #5690 about this.

**Comment by JIoJIaJIu** [comment]
Yes, close it, please
